### PR TITLE
Fix broken link

### PR DESCRIPTION
**Pull request by fernie-cpu**
- [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/master/CONTRIBUTING.md).

#### 1.Describe the changes made:

Replaced broken link on lesson _USING CSS PREPROCESSORS TO SAVE TIME_ under HTML/CSS section
(1st assignment).

Broken link: [Introduction to CSS Preprocessors from Steven Bradley](http://www.vanseodesign.com/css/css-preprocessors/)

EDIT: Instead of replacing the article, I _updated_ the link to use a version from the Web archive, as suggested by @xandora

### CODE REVIEW

**Comment by xandora** [comment]
Medium articles are tricky, as non-paying users only get a certain number of free articles per month before they get paywalled.

We could replace the broken link with a Wayback Machine archive (http://web.archive.org/web/20200430084229/http://www.vanseodesign.com/css/css-preprocessors/), or perhaps a different link that's accessible to anyone via a different blog (https://www.mikestreety.co.uk/blog/an-introduction-to-css-preprocessors/).

**Comment by xandora** [comment]
I also believe we use asterisk based lists throughout the rest of the lessons, so I'm not sure the switch to hyphens is necessary?

File: html_css/preprocessors.md [hunk 1]
```diff
@@ -17,15 +18,16 @@ Preprocessors (aka precompilers) can make your life much easier by eliminating c
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
-1. Read [Introduction to CSS Preprocessors from Steven Bradley](http://www.vanseodesign.com/css/css-preprocessors/)
```
Comment (CatQueenCodes) [inline html_css/preprocessors.md:20]: This link works for me. Can you try it again and let me know.

Comment (fernie-cpu) [reply html_css/preprocessors.md:20]: Oh now it's fine. At the time the whole website was throwing a request error. They fixed it. Sorry about that

**Pull request closed without merging**